src/bindings/rust/wrapper.cpp

```diff
@@ -1824,4 +1823,50 @@ nixl_capi_query_mem(nixl_capi_agent_t agent,
     }
 }
 
+static nixl_capi_status_t
+nixl_capi_status_from_nixl_status(nixl_status_t status) {
+    switch (status) {
+    case NIXL_SUCCESS:
+        return NIXL_CAPI_SUCCESS;
+    case NIXL_IN_PROG:
+        return NIXL_CAPI_IN_PROG;
+    case NIXL_ERR_NO_TELEMETRY:
+        return NIXL_CAPI_ERROR_INVALID_STATE;
```

Comment:
Where does the "ERROR_INVALID_STATE" string come from?
Seems we translate C `NIXL_ERR_NO_TELEMETRY` to `NIXL_CAPI_ERROR_INVALID_STATE`, then to Rust `NixlError::TelemetryNotEnabled`